apps/vor/pkg/classes/store.go

```diff
@@ -1,4 +1,4 @@
-package class
```

Comment:
why change the package name? i think the current name is already good. for consistency purpose

Reply:
yeah, that wasn't really a good call. Reverted now.